module/VuFind/tests/unit-tests/src/VuFindTest/Search/Solr/ResultsTest.php

```diff
@@ -59,6 +60,57 @@ public function testCursorMark(): void
         $this->assertEquals('foo', $results->getCursorMark());
     }
 
+    /**
+     * Test facet translation functionality.
+     *
+     * @return void
+     */
+    public function testFacetTranslations(): void
+    {
+        $mockTranslator = $this->createMock(TranslatorInterface::class);
+        $mockTranslator->expects($this->exactly(2))
```

Comment:
`$this->exactly(2)` should be changed to `$this->once()` when the bug I just noted is fixed... I had to put it in this way to make the test pass for now.